applications/OptimizationApplication/tests/algorithm_tests/analysis_based_tests/algorithm_steepest_descent/optimization_parameters.json

```diff
@@ -0,0 +1,93 @@
+{
+    "problem_data": {
+        "parallel_type": "OpenMP",
+        "echo_level": 0
+    },
+    "model_parts": [
+        {
+            "settings": {
+                "model_part_name": "Structure",
+                "domain_size": 3,
+                "input_filename": "Structure"
+            }
+        }
+    ],
+    "analyses": [
+        {
+            "name": "Structure_static",
+            "type": "stepping_analysis_execution_policy",
+            "settings": {
+                "model_part_names": [
+                    "Structure"
+                ],
+                "analysis_module": "KratosMultiphysics.StructuralMechanicsApplication",
+                "analysis_type": "StructuralMechanicsAnalysis",
+                "analysis_settings": {
+                    "@include_json": "primal_parameters.json"
+                }
+            }
+        }
+    ],
+    "responses": [
```

Comment:
This is not really an analysis-based test, because you use mass. Would it be possible to update the case to optimize for strain energy ?!

Reply:
I have changed the test case to use strain energy